src/components/Player/Livepeer/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Player } from "@livepeer/react";
```

Comment:
The build is failing with `Module not found: Error: Can't resolve '@livepeer/react' in '/vercel/path0/src/components/Player/Livepeer'`. Probably missing adding it to package.json